app/models/cases.py

```diff
@@ -24,7 +23,22 @@ class Case(BaseCase, TableModelMixin, table=True):
     )
 
 
-class CaseRequest(BaseCase):
-    """Request model used to create a new case"""
+class CaseRequest(BaseRequest):
+    case_type: CaseTypes
+    notes: List[CaseNotesRequest] | None
+    people: List[PersonRequest] | None
+    case_tracker: CaseTrackerRequest | None
+    eligibility_outcomes: List[EligibilityOutcomesRequest] | None
+
+    class Meta(BaseRequest.Meta):
+        model = Case
+
+
+class CaseResponse(BaseResponse):
+    notes: List[CaseNotesResponse] | None
+    people: List[PersonResponse] | None
+    case_tracker: CaseTrackerResponse | None
+    eligibility_outcomes: List[EligibilityOutcomesResponse] | None
 
-    pass
+    class Config:
+        orm_mode = True
```

Comment:
Should we set this in BaseResponse rather than on each sub Response class that uses relationships.

Also I believe `orm_mode` was changed to `from_attributes` in Pydantic V2 as per: [https://docs.pydantic.dev/latest/migration/#changes-to-config](https://docs.pydantic.dev/latest/migration/#changes-to-config)

This could be implemented as:

```python
class BaseResponse:
   model_config = ConfigDict(from_attributes=True)
```

Reply:
Thanks @BenMillar-MOJ . I have updated the PR to reflect this now